contrib/guix/manifest.scm

```diff
@@ -195,8 +222,8 @@ chain for " target " development."))
         python-3.7
         ;; Git
         git
-        ;; Native gcc 9 toolchain targeting glibc 2.27
-        (make-gcc-toolchain gcc-9 glibc-2.27))
+        ;; Native gcc 7 toolchain
+        gcc-toolchain-7)
```

Comment:
Why is this changed, seems unrelated to macos, no? Does this break gcc cross builds for our linux archs that need gcc 8 at least?

Reply:
Btw, this is the comment I was referring to: https://github.com/bitcoin/bitcoin/pull/13665#issuecomment-405601228

I presume that guix is using a gcc-7 that has a fix backported?